### PR TITLE
Firefox 60 released

### PR DESCRIPTION
**Pull request by afmenez**
Firefox 60 released today.

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

File: environments.json [hunk 1]
```diff
@@ -921,20 +921,28 @@
     "obsolete": false
   },
   "firefox60": {
-    "full": "Firefox 60 Beta",
+    "full": "Firefox 60",
```
Comment (chicoxyzzy) [inline environments.json]: It should be `Firefox 60  ESR`

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]
The full name should be Firefox 60 ESR. Otherwise LGTM

**Comment by afmenez** [comment]
I believe this may be confusing.  On every ESR release before this one, we only added the ESR suffix when the next version came out.

**Comment by chicoxyzzy** [comment]
Hm, I don't understand what exactly is confusing. Firefox 60 is ESR so we should reflect this in compatibility table. 

https://www.mozilla.org/en-US/firefox/organizations/

image

**Comment by afmenez** [comment]
You see, this is exactly the confusing part. As you can see in the graph, Firefox ESR (in yellow) are separate releases from the mainline (in green).  Firefox 60 IS not an ESR release, it HAS an ESR release.  Most people are in the main release train and don't even know what the ESR is, and that's why it may be confusing.
The rendering engine is identical for the normal and ESR releases, so I don't believe we should have 2 separate entries.

**Comment by chicoxyzzy** [comment]
We don't reflect rendering engine features in our table. Regular release and ESR have the same SpiderMonkey version. Sometimes some features could be disabled in ESR version. Actually it happened once with WebAssembly (it was disabled in 52 ESR) which is also not reflected in table. If something like this will happen in future, we can add note about this.

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**